<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogchange</class>
 <widget class="QDialog" name="Dialogchange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>787</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>787</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ChangeInfo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="picture.qrc">
    <normaloff>:/travelsystem/success</normaloff>:/travelsystem/success</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255,255,255)</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-29</x>
     <y>319</y>
     <width>821</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(135, 139, 141)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="strategyEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>42</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;
border-radius:5px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>42</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(97, 99, 101);
font: 9pt &quot;Arial&quot;;
</string>
   </property>
   <property name="text">
    <string>New plan:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>169</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(97, 99, 101);
font: 9pt &quot;Arial&quot;;
</string>
   </property>
   <property name="text">
    <string>Pass-by citys:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="toEdit">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>83</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;
border-radius:5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>211</y>
     <width>701</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;
border-radius:5px;
</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>342</y>
     <width>162</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(62, 175, 236);
font: 10pt &quot;Consolas&quot;;
border:2px  ;
border-color:rgb(24, 186, 210);
border-radius:30px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>124</y>
     <width>136</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(97, 99, 101);
font: 9pt &quot;Arial&quot;;
</string>
   </property>
   <property name="text">
    <string>Time limited:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tlEdit">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>124</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;
border-radius:5px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>83</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(97, 99, 101);
font: 9pt &quot;Arial&quot;;
</string>
   </property>
   <property name="text">
    <string>New destination:</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmBg">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>166</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(57, 154, 221);
border-radius:33px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>confirmBg</zorder>
  <zorder>widget</zorder>
  <zorder>strategyEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>label_8</zorder>
  <zorder>toEdit</zorder>
  <zorder>textEdit</zorder>
  <zorder>confirm</zorder>
  <zorder>label_7</zorder>
  <zorder>tlEdit</zorder>
  <zorder>label_4</zorder>
 </widget>
 <tabstops>
  <tabstop>strategyEdit</tabstop>
  <tabstop>toEdit</tabstop>
  <tabstop>tlEdit</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>confirm</tabstop>
 </tabstops>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
